README.md

```diff
@@ -34,17 +40,47 @@ dev_dependencies:
   flappy_translator: 
 ```
 
+### Define Settings
+
+Settings for *flappy_translator* can be optionally set in your project's *pubspec.yaml* file:
+
+```yaml
+flappy_translator:
+  input_file_path: "test.csv"
+  output_dir: "lib"
+  file_name: "i18n"
+  class_name: "I18n"
+  delimiter: ","
+  start_index: 1
+  depend_on_context: true
+  use_single_quotes: false
+```
+
+| Setting           | Default | Description                                                                     |
+| ----------------- | ------- | ------------------------------------------------------------------------------- |
+| input_file_path   | N/A     | A path to the input CSV file.                                                   |
+| output_dir        | lib     | A directory to generate the output file.                                        |
+| file_name         | i18n    | A filename for the generated file.                                              |
+| class_name        | I18n    | A class name for the generated file.                                            |
+| delimiter         | ,       | A delimited to separate columns in the input CSV file.                          |
+| start_index       | 1       | The column index where translations begin (i.e. column index of main language.) |
+| depend_on_context | true    | Whether the generated localizations should depend on *BuildContext*             |
```

Comment:
Maybe we should explain a little bit more the fact that, if not depending on BuildContext, the strings will not be updated on phone settings changes

Reply:
The phone settings change for me, even when BuildContext isn't depended upon. 

Open App -> Background App -> Go to settings -> Change OS Language from FR to EN -> Open App -> App is in EN and not FR

This worked for me on iOS, however I will do some more testing.

BTW, the following should be added to Info.plist, otherwise iOS doesn't know what languages the app supports:

```
<key>CFBundleLocalizations</key>
<array>
    <string>fr</string>
    <string>en</string>
    <string>es</string>
    <string>de</string>
</array>
```

Reply:
@ThomasEcalle I have updated the example app and added some info in README about Info.plist.